add_to_app/README.md

```diff
@@ -36,7 +36,7 @@ you're building for both iOS and Android, with both toolchains installed):
   flutter build aar
 
   # For iOS builds:
-  flutter build ios-framework --output=../ios_using_prebuilt_module/Flutter
+  flutter build ios-framework --xcframework --output=../ios_using_prebuilt_module/Flutter
```

Comment:
Since we're targeting stable here, I assume everything still works corretly on stable?

Reply:
Yup `--xcframework` is on stable.  With this change it outputs the universal `.framework`s on stable but it's just not consumed by the host app.  On master it doesn't output the universal frameworks.

Reply:
And it worked against stable in the CI task.